Sources/DistributedActors/ClusterSystem.swift

```diff
@@ -331,6 +331,13 @@ public class ClusterSystem: DistributedActorSystem, @unchecked Sendable {
             _ = self._receptionistStore.storeIfNilThenLoad(receptionist)
         }
 
+        // downing strategy (automatic downing)
+        await _Props.$forSpawn.withValue(DowningStrategyShell.props) {
```

Comment:
Not sure about move it here

Reply:
Hmm it probably is fine; previously the reason to be in the shell was to become a child of the cluster actor but we don't have child actors anymore so I think this may be fine. If tests are happy we're happy 😃

Reply:
The entire props dance were able to simplify very soon by the way - the actor system can notice the static props property and pick it up etc